content/en/docs/release-notes/release-notes-1.4.md

```diff
@@ -34,6 +32,220 @@ We have deprecated the following cert-manager APIs:
 - `acme.cert-manager.io/v1alpha3`
 - `acme.cert-manager.io/v1beta1`
 
-These APIs will be removed in cert-manager `v1.6.0`.
-If you have a cert-manager installation that is using or has previously used these deprecated APIs you may need to upgrade your cert-manager custom resources and CRDs. This needs to be done before upgrading to cert-manager `v1.6.0`.
-See cert-manager [docs](../../installation/upgrading/remove-deprecated-apis/#upgrading-existing-cert-manager-resources) for more detailed upgrade instructions.
+These APIs will be removed in cert-manager 1.6.
+
+{{% pageinfo color="warning" %}}
+
+⛔️  If you have a cert-manager installation that is using or has previously used
+these deprecated APIs, you may need to upgrade your cert-manager custom
+resources and CRDs.
+
+This needs to be done before upgrading to cert-manager 1.6. See cert-manager
+[docs](../../installation/upgrading/remove-deprecated-apis/#upgrading-existing-cert-manager-resources)
+for more detailed upgrade instructions.
+
+{{% /pageinfo %}}
+
+## Helm chart: `securityContext` defaults to non-root
+
+The Helm chart [now](https://github.com/jetstack/cert-manager/pull/4036) follows
+the current Pod hardening best practices as defined by the Kyverno [pod-security
+restricted](https://kyverno.io/policies/pod-security/#restricted) policy.
+
+To pass the validation, the controller, webbook, and cainjector Pods are now
+running as non-root:
+
+```yaml
+apiVersion: v1
+kind: Pod
+spec:
+  securityContext:
+    runAsNonRoot: true
+```
+
+{{% pageinfo color="warning" %}}
+
+⛔️  If you are using custom containers that run as root with the Helm chart, you
+will need to set this back to `false`.
+
+{{% /pageinfo %}}
+
+## CA Issuer and `ca.crt`
+
+The CA issuer [now](https://github.com/jetstack/cert-manager/pull/3865) attempts
+to store the root CA instead of the issuing CA into the `ca.crt` field for
+issued certificates. All of the information which was previously available is
+still available: the intermediate should appear as part of the chain in
+`tls.crt`.
```

Comment:
I suggest removing the `now` PR link.
And if you want to add PR links, do it at the end of each note as `Implemented in cert-manager PR#3865` (or something like that)